packages/cli/src/tools/envinfo.ts

```diff
@@ -16,6 +17,17 @@ async function getEnvironmentInfo(
 ): Promise<string | EnvironmentInfo> {
   const options = {json, showNotFound: true};
 
+  let packages = ['react', 'react-native', '@react-native-community/cli'];
+
+  const outOfTreePlatforms: {[key: string]: string} = {
```

Comment:
Can we rely on TS interference here? We should be able to avoid the "any" on line 26